instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/AwsSdkTracingBuilder.java

```diff
@@ -6,16 +6,20 @@
 package io.opentelemetry.instrumentation.awssdk.v1_11;
 
 import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.config.Config;
 
 /** A builder of {@link AwsSdkTracing}. */
 public class AwsSdkTracingBuilder {
+  private static final boolean CAPTURE_EXPERIMENTAL_SPAN_ATTRIBUTES_DEFAULT =
+      Config.get().getBoolean("otel.instrumentation.aws-sdk.experimental-span-attributes", false);
 
   private final OpenTelemetry openTelemetry;
 
   private boolean captureExperimentalSpanAttributes;
 
   AwsSdkTracingBuilder(OpenTelemetry openTelemetry) {
     this.openTelemetry = openTelemetry;
+    this.captureExperimentalSpanAttributes = CAPTURE_EXPERIMENTAL_SPAN_ATTRIBUTES_DEFAULT;
```

Comment:
other library instrumentations only allow programmatic configuration of `captureExperimentalSpanAttributes`, I'd suggest doing the same here for consistency (setting this from the autoconfigure module), and opening an issue to discuss doing this across all instrumentations if there's interest

Reply:
Hi @trask ,

The `captureExperimentalSpanAttributes` could already be configured by these system properties/environment variables.  I only moved this mechanism to make sure this can be shared between the java-agent and the autoconfigure library.  
I understand that the default for the builder are now also changing based on these variables.
Would it be a solution if I add an extra method to the `AwsSdkTracingBuilder`, which is then called by both the java-agent as the autoconfigure class?
Or do you prefer the PR where I copy this behaviour to all other classes which have these `Config` variables for the `java-agent`?

Reply:
> Would it be a solution if I add an extra method to the `AwsSdkTracingBuilder`, which is then called by both the java-agent as the autoconfigure class?

`AwsSdkTracingBuilder` already has `setCaptureExperimentalSpanAttributes`, can you call this from both javaagent and autoconfigure modules? I think that would be most consistent with other instrumentations currently.

Reply:
@trask I tried to update it, while still keeping the Config parameter in one place for v1.1.
Let me know what you think.